cluster-state-service/handler/api/v1/taskapis.go

```diff
@@ -24,19 +24,22 @@ import (
 	"github.com/blox/blox/cluster-state-service/handler/store"
 	"github.com/blox/blox/cluster-state-service/handler/types"
 	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
 )
 
 const (
 	taskARNKey     = "arn"
 	taskClusterKey = "cluster"
 
-	taskStatusFilter  = "status"
-	taskClusterFilter = "cluster"
+	taskStatusFilter    = "status"
+	taskClusterFilter   = "cluster"
+	taskStartedByFilter = "startedBy"
```

Comment:
@kylbarnes Off the top of my head, I know Facebook and Yelp uniformly use snake_case for query param keys